operator/pkg/translate/translate.go

```diff
@@ -522,6 +522,11 @@ func (t *Translator) setComponentProperties(root map[string]interface{}, iop *v1
 		}
 	}
 
+	rootNamespace := name.RootNamespaceForMeshConfig(iop)
+	if err := tpath.WriteNode(root, util.PathFromString(name.MeshConfigKey+"."+name.RootNamespaceKey), rootNamespace); err != nil {
```

Comment:
Why do we need this code if the logic is in the helm chart?

Reply:
removed additional code for operator.